DXRando/DeusEx/Classes/EndgameQuoteManager.uc

```diff
@@ -6,7 +6,7 @@ struct EndQuote
     var string attribution;
 };
 
-var EndQuote quotes[140];
+var EndQuote quotes[148];
```

Comment:
at some point we should probably just ditch the array and load function, and just do a retrieve function in the style of `if(r--==0)`